tooling/bin/generate-ssi-package.sh

```diff
@@ -23,47 +23,97 @@ for architecture in "${architectures[@]}"; do
     root=$tmp_folder_final/$architecture/dd-library-php-ssi
     gnu=$root/linux-gnu
     musl=$root/linux-musl
-    trace=$root/trace
 
     # Starting from a clean folder
     rm -rf ${tmp_folder}
-    mkdir -p ${gnu}/loader ${musl}/loader ${trace}
 
     ########################
     # Loader
     ########################
-    ln libddtrace_php_${architecture}.so ${gnu}/loader/libddtrace_php.so
-    ln libddtrace_php_${architecture}-alpine.so ${musl}/loader/libddtrace_php.so
 
-    ln dd_library_loader-${architecture}-linux-gnu.so ${gnu}/loader/dd_library_loader.so
-    ln dd_library_loader-${architecture}-linux-musl.so ${musl}/loader/dd_library_loader.so
+    mkdir -p ${gnu}/loader ${musl}/loader
+
+    cp libddtrace_php_${architecture}.so ${gnu}/loader/libddtrace_php.so
```

Comment:
Is there a reason you switched `ln` (hard links) to `cp` (copy)?
This was done for performance reasons in packaging CI step.

Reply:
I suppose it's because of the stripping, but maybe use a bash function for that:

```
function stripto() {
   $(if [[ "${architecture}" == "aarch64" ]]; then echo aarch64-linux-gnu-; fi)-strip -o "$2" "$1"
}
```

Reply:
Yes, that's due to the stripping of the .so

Reply:
Updated